grafana/scylla-os.template.json

```diff
@@ -617,28 +632,11 @@
                     "type": "custom"
                 },
                 {
-                    "allValue": null,
-                    "current": {
-                        "isNone": true,
-                        "text": "None",
-                        "value": ""
-                    },
-                    "datasource": "prometheus",
-                    "hide": 0,
-                    "includeAll": false,
+                    "class": "template_variable_all",
                     "label": null,
-                    "multi": true,
                     "name": "monitor_disk",
-                    "options": [],
                     "query": "node_disk_read_bytes_total",
-                    "refresh": 2,
-                    "regex": "/.*device=\"([^\\\"]*)\".*/",
-                    "sort": 0,
-                    "tagValuesQuery": "",
-                    "tags": [],
-                    "tagsQuery": "",
-                    "type": "query",
-                    "useTags": false
+                    "regex": "/.*device=\"([^\\\"]*)\".*/"
```

Comment:
So it'd show both md0 and the individual disks?

Reply:
The change is to show all disks by default, instead of just one